camel-prod-maven-plugin/src/main/java/org/l2x6/cq/camel/maven/prod/CamelProdExcludesMojo.java

```diff
@@ -487,13 +501,16 @@ void handleExcludedTargetDirectories(final Path basePath, final MavenSourceTree
                     final String originalBuildDir = project.getBuild().getDirectory();
                     project.getBuild().setDirectory(moduleBaseDir.resolve("target").toString());
                     try {
+                        project.getProperties().setProperty("camel.pmp.json-directory", "catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/components");
```

Comment:
I do not see this `camel.pmp.json-directory` being used anywhere in Camel main source tree. What is it for?